src/aws.js

```diff
@@ -32,3 +33,23 @@ export const clientConfig = {
         httpsAgent: agent,
     }),
 };
+
+/**
+ * Attempt to identify the IAM user name behind the current AWS credentials
+ * (useful for debugging / server start-up messages). Some external users of
+ * the nextstrain server may be using non-AWS services, but these may still be
+ * compatible with STS so we make a best-effort attempt here.
+ */
+export async function getAwsId() {
+  const client = new STSClient({ region: AWS_REGION });
```

Comment:
Sorry for not noticing on the first round, but this should be using the shared global client config defined immediately above, e.g.

```suggestion
  const client = new STSClient(clientConfig);
```

Reply:
Changed in repush.